bot/exts/moderation/incidents.py

```diff
@@ -130,6 +139,39 @@ def has_signals(message: discord.Message) -> bool:
     return ALL_SIGNALS.issubset(own_reactions(message))
 
 
+async def make_message_link_embed(ctx: Context, message_link: str) -> discord.Embed:
+    """
+    Create an embedded representation of the discord message link contained in the incident report.
+
+    The Embed would contain the following information -->
+        Author: @Jason Terror ♦ (736234578745884682)
+        Channel: Special/#bot-commands (814190307980607493)
+        Content: This is a very important message!
+    """
+    embed = discord.Embed()
+
+    try:
+        message_convert_object = MessageConverter()
+        message = await message_convert_object.convert(ctx, message_link)
+
+    except discord.DiscordException as e:
+        embed.title = str(e)
+        embed.colour = Colours.soft_red
```

Comment:
Errors shouldn't be sent in `#incidents`. Instead, they should be logged with `log.exception` and the function should return.